pkg/utils/crtl_utils.go

```diff
@@ -77,7 +77,7 @@ func Now() *metav1.Time {
 	now := metav1.Now()
 	return &now
 }
-
+// Determine whether the string array contains a specific string, return true if contains the string and return false if not.
```

Comment:
ContainsString determines whether the string array contains a specific string, return true if contains the string and return false if not.